### PR TITLE
Version 3.0

### PR DESCRIPTION
**Pull request by OpenSourceSimon**
After 5-6 months it's finally here! Version 3.0 includes loads of new features and bug fixes, here's a quick list:

- Fix for TikTok TTS, which now requires a sessionid
- Story mode feature so it can read the post description
- Added custom resolution, landscape videos are possible
- Added AI random posts filtering
- More features to the GUI
- Create a thumbnail

### CODE REVIEW

**Comment by ayoubmca111** [comment]
synonyms for work of art

    artwork.
    composition.
    piece.
    drawing.
    object of art.
    objet d'art.
    study.

**Review by LevaniVashadze — APPROVED** [review]
cool

**Comment by CJthisis** [comment]
Finally!

File: fonts/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```
Comment (callumio) [inline fonts/LICENSE.txt:2]: A note about the used fonts and their license(s) should also be included in the footer of our README.

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/playwright
+FROM python:3.10.9-slim
```
Comment (callumio) [inline Dockerfile:1]: Think I'll rewrite this Dockerfile at some point

**Review by callumio — COMMENTED** [review]
I don't really have time to go through all of this in depth or test, but overall it looks good.

Can you confirm that :
- all the code has been formatted to match the style guideline?
- `requirements.txt` is up to date

The entire project would benefit greatly from better typehinting, but that can be done at a later date.

---

P.S. the mp3's really should never have been commited to git and rather downloaded as needed, but they're in there now so no point removing them.

**Comment by JasonLovesDoggo** [comment]
I can't say anything except for what Callum said

**Review by callumio — APPROVED** [review]

**Comment by Ika1009** [comment]
Hey, where can I get the TikTok sessionid?

**Pull request merged**